app/views/pages/home.html.erb

```diff
@@ -1,10 +1,4 @@
-
-
-
-
-<body>
```

Comment:
@vlopergolo I am deleting this body tag as it's already defined in the file "application.erb"